modules/peridynamics/doc/content/source/userobjects/RandomizeCriticalValuePD.md

```diff
@@ -0,0 +1,13 @@
+# Randomize Critical Value UserObject
+
+## Description
+
+UserObject `RandomizeCriticalValuePD` is used to assign randomized values for the `critical_variable` auxiliary variable in fracture modeling. The generated values can either be uniform or follow normal distribution.
+
+Using GeneralUserObject, a unique sampling is used for all the bonds, i.e., the critical values of the bonds follow a normal distribution. However, using AuxKernel, bonds are partitioned into different processors and each processor performs its own sampling for bonds belongs this processor only. As a result, the overall distribution for the critical value of bonds across all processors may not follow the intended normal distribution. (The sum of two independent normally distributed random variables is normal, but with its mean being the sum of the two means, and its variance being the sum of the two variances.)
```

Comment:
Have you tried using RandomIC?  It can generate randomized variables that follow distributions defined by any of the many `Distribution` classes in MOOSE, and they are parallel consistent.  I suspect that you could use it instead of this class.

Reply:
I removed this class and we can use RandomIC instead.